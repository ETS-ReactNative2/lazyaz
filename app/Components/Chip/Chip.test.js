import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Chip } from './index';

test('Chip component renders correctly', () => {
  const props = {
    places: 0,
  };
  const tree = renderer.create(<Chip {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
