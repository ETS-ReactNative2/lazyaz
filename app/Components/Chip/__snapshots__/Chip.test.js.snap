// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip component renders correctly 1`] = `
<View>
  <Text>
    0 places
  </Text>
</View>
`;
